spec/features/features_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
```

Comment:
We don`t need to require rails_helper.
I added it to the .rspec file

Reply:
I am not sure that we need to require rails_helper in every spec, because we have at least unit and feature specs, which have different configurations and should require different helper files

Reply:
Do you have two rails_helper.rb files?

Reply:
@AZatsepa, no a have features_helper file, with feature specs configurations. In my thought, we do not have to put configurations for both, feature and unit specs in one rails_helper. Instead we should put additional configs for feature specs into another configurations file (features_helper), for more readability and less useless module inclusions in unit specs

Reply:
I commented your requirement and all specs work correctly. Looks like this is a duplication.

Reply:
@AZatsepa, As a told, I think that we do not need to include feature configurations into every unit spec

Reply:
@AZatsepa, also we do not need a duplication, so, in my opinion, at this moment, it looks good